### DISCUSSION TITLE
download multiple users with yaml file?

### DISCUSSION THREAD

**Discussion by bookish1303** [Serene-Arc/bulk-downloader-for-reddit#958] [Q&A]
Hi there:

Just trying this out, and attempting to download from multiple user profiles by putting the list of users into the options YAML file.

Here's what my test yaml file looks like:

```
no_dupes: true
folder_scheme: "{REDDITOR}"
limit: 10
sort: top
user:
  - user1
  - user2
```

Running bdfr with command `D:\Downloads\bdfr' --opts opts.yml -v` returns
```
[2024-05-26 03:53:33,628 - bdfr.connector - DEBUG] - Disabling the following modules:
[2024-05-26 03:53:33,639 - bdfr.connector - DEBUG] - Using unauthenticated Reddit instance
[2024-05-26 03:53:33,641 - root - INFO] - Program complete
```

I figured that it would be a matter of taking the sample yaml file and changing from `subreddit` to `user`, but it seems like that's not the case. There's an [earlier discussion](https://github.com/aliparlakci/bulk-downloader-for-reddit/discussions/897) but it seems like it didn't go anywhere. Does anyone have any advice on the correct syntax to use?

Thanks!

**Comment by superrob1500** [comment]
I don't know the answer with the YAML method, but an alternate is to keep users in text lists. With this method then you can make a Powershell script that reads the list and passes each user to BDFR similar to the Linux xargs command suggested in the README and the thread linked. 

```
$users = Get-Content PATH_TO_USER_LIST | ForEach-Object { "--user $_" }
$command = "& bdfr download D:\Downloads\bdfr --submitted --opts opts.yml $users" 
Invoke-Expression $command
```

inside the `$command` variable you can add all the options you want to BDFR. It's not as clean as a single command like with xargs but it should work. Such is the Windows life.
